examples/docker-build/dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:latest as builder
```

Comment:
I think `node:latest` is not alpine based. Can we use `node:alpine` here as well?

Reply:
Since this dockerfile is multistage, `node:latest` won't affect any kind of build. Final build will always be small because of `node:latest-alpine`. Moreover I would suggest using `node:latest` image as a builder image like how we are using here because modules like `image-webpack-loader` have native dependencies, won't compile directly in `node:latest-alpine` image(requires other packages).

Reply:
My reasoning is more because of that there may be native modules that are incompatible with alpine. We install production `node_modules` in a Debian based container and then run it in an alpine based one. Maybe that would be better to move second `yarn install` step to the second stage and also remove yarn cache but it makes our example more complex. So we can ignore it for now and later use better documentation to explain such in-depth details.